src/database/db_build.sql

```diff
@@ -0,0 +1,29 @@
+BEGIN;
+    DROP TABLE IF EXISTS users,articles,bookings,services CASCADE;
+CREATE TABLE
+    IF NOT EXISTS services(
```

Comment:
this is redundant a bit no? cause on line 2 you are saying if it exists remove it, so why check if it doesn't exist?